integrations/extensions/docs/elasticsearch-install-and-setup/how_to_index_pdf_and_office_documents_elasticsearch.md

```diff
@@ -0,0 +1,217 @@
+
+You can now run the fscrawler to ingest your documents.
+
+NOTE: If the updated date of the documents are older than the current date, you would have to follow instructions as mentioned [here](https://fscrawler.readthedocs.io/en/latest/user/tips.html#moving-files-to-a-watched-directory) to ensure the fscrawler is able to pick it up for indexing.
+
+Next, navigate to the `fscrawler-docker-config` directory and run:
+
+```
+docker-compose up -d
+```
+
+This will start the `fscrawler` app and begin to ingest the documents from the directory it was pointed at. 
+
+a. If you have installed Kibana locally following [ICD-elasticsearch-install-and-setup](./ICD_Elasticsearch_install_and_setup.md#step-2-set-up-kibana-to-connect-to-elasticsearch),  then you can verify this by bringing up Kibana. Once Kibana is up and running successfully:
+
+	1. Navigate to http://localhost:5601/app/enterprise_search/elasticsearch
+	2. Click on "Indices" and click on your chosen index. 
+		* The "Overview" tab will be updated with the number of documents indexed or being indexed. 
+		* You can verify the index to make sure the crawler ingested all documents successfully
+
+b. If not, you can use the following to check the document ingestion count as well which will be displayed in the `count` field of the output from the below API request
+
+```bash
+curl -X GET "${ES_URL}/${ES_INDEX_NAME}/_count?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+-H "Content-Type: application/json" --cacert "${ES_CACERT}"
+```
+
+OPTIONAL: Once all documentss are indexed, you can stop the `fscrawler` app or if you would like to, you can leave it running to keep the filesystem in sync if new documents are added or old ones removed. To stop the app , run the following:
+
+```
+docker-compose down
+```
+
+Your documents are now available in the index, ready for searching and querying. Follow the steps outlined below to use this index in a RAG based setup with Watson Assistant. 
+
+### Step 5: Connecting Watson Assistant to Elasticsearch for Conversational Search
+
+Follow the steps outlined in guide to [connect your assistant to elasticsearch and watsonx using custom extensions](../../starter-kits/language-model-conversational-search#example-1-connect-your-assistant-to-elasticsearch-and-watsonx-via-custom-extensions) 
+
+> ⛔️
+> **Caution**  
+> 
+> * After uploading the sample workspace JSON file to your Assistant, make sure to update the session variable `has_inner_hits` to `True`. This will ensure that the appropriate `query_body` will be used for your index. 
+> 
+> * Remember to set `_source` to `query_source` session variable when setting up your Elasticsearch extension. By limiting the fields available in the `_source`, we can limit the length of the query response, potentially avoiding the 500 error that may be encountered due to length limits.
+
+#### Example usage:
+
+Here is an example of how to use the `Search` action for this starter kit conversational search example:
```

Comment:
Does this example align with the content that is suggested to use as an example?  The suggested example PDF is doc for WA, so that question seems out of place

Reply:
Good point, updated the example docs and the example shown on assistant

Reply:
Hi Sneha, I thought we were going to use the WA docs PDF as an example so that it is consistent with the other guides. The example link you provided points to [a box folder](https://ibm.ent.box.com/s/l2rhvajs6jasz2rqla7eakz7uml8oljs/folder/230228063107). Is it publicly accessible? If it's not, we shouldn't use it.

Reply:
I checked with Nic regarding using the documents from that folder. The `Lendyr` documents and `Lendyr` website is something we have been using in some of our other guides( not Elastic guides but other WA guides ) , so I don't see an issue with suggesting that, especially because they are smaller documents and there are multiple documents as well.

Reply:
Done, seems like it is available from https://assistant-demo.global.assistant.test.watson.appdomain.cloud/test/search-crawls/pdfs/lendyr-docs/contents.html , however that may change in the future so I included a couple docs with this PR itself